app/src/components/Shared/LanguageSelector.jsx

```diff
@@ -16,46 +17,56 @@ class LanguageSelector extends Component {
   };
 
   componentDidMount = () => {
-    if (window.bablic !== undefined) {
-      const languages = window.bablic.languages.get();
-      const currentLng = window.bablic.getLocale();
+    if (window.bablic !== undefined && bablic.loaded) {
```

Comment:
`window.bablic.loaded`
In fact I think we should have an eslint rule for this.

Reply:
```suggestion
    if (window.bablic !== undefined && window.bablic.loaded) {
```